docs/azure/dotnet-dev-env-checklist.md

```diff
@@ -0,0 +1,51 @@
+---
+title: .NET Development on Azure Configuration Checklist
+description: Provides a quick summary of all the tools you should have installed to do .net development with Azure
+ms.date: 1/1/2021
+ms.topic: conceptual
+ms.custom: devx-track-dotnet
+ms.author: daberry
+author: daberry
+---
+
+# .NET on Azure development environment checklist
+
+This checklist is provided to help you make sure you have your development environment correctly configured for .NET development with Azure
+
+## Create an Azure account
+
+To access Azure services or run applications in Azure, you need an Azure account.
+
+* If you are a Visual Studio subscriber, you have monthly [free Azure credits](https://azure.microsoft.com/pricing/member-offers/credit-for-visual-studio-subscribers/) available to you every month
+* [Create a free Azure account](https://azure.microsoft.com/free/dotnet/) and receive $200 in credits and select services free for 12 months
+* Use an account assigned to you by your company's Azure administrator
+
+## Configure your IDE
+
+Popular tools like Visual Studio and Visual Studio Code have extensions available to let you work with Azure right from your IDE.
+
+* [Configure Visual Studio](./configure-visual-studio) for .NET development using Azure
+* [Configure Visual Studio Code](./configure-vs-code) for .NET Development using Azure
+
+## Install the Azure CLI
+
+In addition to using the Azure portal, you will want to install the Azure CLI to create and manage Azure resources.
+
+* [Install the Azure CLI for Windows](/cli/azure/install-azure-cli-windows?tabs=azure-cli))
+* [Install the Azure CLI for macOS](/cli/azure/install-azure-cli-macos)
+* [Install the Azure CLI for Linux](/cli/azure/install-azure-cli-linux)
+
+## Install additional tools and utilities
+
+These additional tools are designed to make you more productive when working with Azure.
+
+* [Install Azure PowerShell](/powershell/azure/install-az-ps) if you plan on using PowerShell scripts to create and manage Azure resources
+* [Install Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) to upload, download, and manage data in Azure storage resources including blobs, queues, tables, and CosmosDB.
```

Comment:
```suggestion
* [Install Azure Storage Explorer](https://azure.microsoft.com/features/storage-explorer/) to upload, download, and manage data in Azure storage resources including blobs, queues, tables, and CosmosDB.
```

Also, does CosmosDB support in Storage Explorer include Mongo? Or just SQL API (or whatever they call DocumentDB these days)?